.circleci/config.yml

```
@@ -214,6 +214,16 @@ jobs:
             PRE_RELEASE=${CIRCLE_TAG/${CIRCLE_TAG%-rc[0-9]*}/}
             github-release delete -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -t ${CIRCLE_TAG} 2>/dev/null ||:
             ./hack/release-notes.sh ${CIRCLE_TAG} | github-release release ${PRE_RELEASE:+-p} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -t ${CIRCLE_TAG} -d -
+
+            # wait for github to report the release before attaching the artifacts
+            try=0
+            max_tries=5
+            while [ ${try} -lt ${max_tries} ]; do
+              github-release info -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -t ${CIRCLE_TAG} >/dev/null && break
+              try=$(expr ${try} + 1)
+              sleep 1
+            done
```

Tip: No need to increment variables to do something n-times.

```suggestion
            for _ in $(seq 1 5); do
              github-release info -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -t ${CIRCLE_TAG} >/dev/null && break
              sleep 1
            done
```